doc/source/getting_started/install.rst

```diff
@@ -334,7 +334,8 @@ lxml                      4.8.0              xml             XML parser for read
 SQL databases
 ^^^^^^^^^^^^^
 
-Installable with ``pip install "pandas[postgresql, mysql, sql-other]"``.
+Traditional ODBC drivers are Installable with ``pip install "pandas[postgresql, mysql, sql-other]"``.  ADBC drivers
```

Comment:
```suggestion
Traditional drivers are installable with ``pip install "pandas[postgresql, mysql, sql-other]"``.  ADBC drivers
```

The traditional ones are not necessarily ODBC. We could say they are "DBAPI" drivers, but also the ADBC drivers give a DBAPI compliant interface, so let's just leave it at "traditional"?